pydantic/decorator.py

```diff
@@ -6,7 +6,7 @@
 
 from ._internal import _typing_extra, _utils
 from .config import Extra, get_config
-from .decorators import validator
+from .decorators import field_validator
```

Comment:
Do we need both decorator.py _and_ decorators.py? Lol. Maybe changing that isn't a this-PR thing haha.

Reply:
I definitely think we don't need both, but like you say not something to change in this PR.